app/models/card.rb

```diff
@@ -1,7 +1,21 @@
 class Card < ActiveRecord::Base
-  before_create :set_review
+  before_validation :set_review
+  validate :check_original_translated_equal
+  validates :original, presence: true
+  validates :translated, presence: true
+  validates :review, presence: true
 
-  def set_review
-    self.review = Time.now + 3.days
-  end
+  protected
+    def set_review
+      self.review = Time.now + 3.days
+    end
+
+    def check_original_translated_equal
```

Comment:
Assignment Branch Condition size for check_original_translated_equal is too high. [19.03/15]<br>Inconsistent indentation detected.